doc/versions.json

```diff
@@ -71,6 +72,12 @@
       "zephyr": "2.6.0-rc1",
       "mcuboot": "1.7.99",
       "nrfx": "2.5"
+    },
+	"1.5.2": {
```

Comment:
alignment here is a bit off but this is the case for all 1.5.x versions so probably not an issue